src/Discord/Builders/CommandAttributes.php

```diff
@@ -241,7 +241,7 @@ public function removeOption(Option $option): self
             throw new \DomainException('Only CHAT_INPUT Command type can have option.');
         }
 
-        if (($idx = array_search($option, $this->option)) !== null) {
+        if (isset($this->options) && ($idx = array_search($option, $this->options)) !== null) {
```

Comment:
shouldn't that be ``false`` instead of ``null`` ?
as per array_search docs (https://www.php.net/manual/en/function.array-search.php#refsect1-function.array-search-returnvalues)